setup.py

```diff
@@ -57,10 +57,13 @@ def multiglob(*globs):
     ('share/applications', multiglob('*.desktop')),
     ('share/metainfo', multiglob('*.appdata.xml')),
     ('share/pronterface', multiglob('images/*.png',
-                                    'images/*.svg',
-                                    'locale/*/LC_MESSAGES/*.mo')),
+                                    'images/*.svg')),
```

Comment:
Looks like this is our problem and images need to go into share/pronterface/images instead of /share/pronterface